src/redist/targets/GenerateLayout.targets

```diff
@@ -166,8 +167,15 @@
         <RelativeLayoutPath></RelativeLayoutPath>
       </BundledLayoutComponent>
 
-      <!-- The AspNetTargetingPackInstallerFile for PKG types does not exist at this time.
-           https://github.com/aspnet/AspNetCore/issues/8806 when resolved, change the 'Condition' below. -->
+      <!-- The AspNetTargetingPackInstallerFile for PKG types does not exist at this time; we are using the archive.
+           https://github.com/aspnet/AspNetCore/issues/8806 -->
```

Comment:
My understanding from this morning is that we are doing the same thing as this for the ASP.NET shared framework in pkg? Is that correct. Is there a bug tracking that as well? There is no point in fixing targeting pack and not fixing shared framework. It's unclear what customer value we are missing by doing it this way for pkg.

Reply:
Please refer to the email thread on this topic. @leecow and @dagood  have outlined a few business reasons for publishing PKG installers from ASP.NET.